dashboards-observability/public/components/custom_panels/custom_panel_view.tsx

```diff
@@ -678,15 +680,34 @@ export const CustomPanelView = ({
               </EuiFlexItem>
               {appPanel && (
```

Comment:
Will keeping this as `&&` restrict the edit button to be run only in app-panels?

Reply:
There is another one above that includes the edit button, etc. with the condition `appPanel ||`. I had to do them separately because they are in different places. `appPanel` will not need a header area.

Reply:
👍🏽  Got it we can create a more modular header for panels later.